backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -179,12 +179,36 @@ export class ContributionResolver {
   async listAllContributions(
     @Args()
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
+    @Arg('statusFilter', () => [ContributionStatus], { nullable: true })
+    statusFilters: ContributionStatus[] | null,
   ): Promise<ContributionListResult> {
+    const where: {
+      contributionStatus?: FindOperator<string> | null
+    } = {}
+    const typeStatus = Object.values(ContributionStatus)
+    if (statusFilters !== null) {
+      const statusFilterArray = []
+      const length = statusFilters.length
+      let i = 0
+      for (i; i < length; i++) {
+        const statusFilter = statusFilters[i]
+        if (!typeStatus.includes(statusFilter)) {
+          logger.error(`Nicht definierter Status wurde gesendet: ${statusFilters[i]}`)
```

Comment:
Can this even happen? Does the schema throw?

Reply:
`"Enum \"ContributionStatus\" cannot represent non-enum value: \"NOT_VALID\".",`

Reply:
`Value \"PENDIG\" does not exist in \"ContributionStatus\" enum. Did you mean the enum value \"PENDING\"?`